apps/frontend/src/components/template/FapReviewTemplatesTable.tsx

```diff
@@ -0,0 +1,208 @@
+import MaterialTable, { Column } from '@material-table/core';
+import Button from '@mui/material/Button';
+import Link from '@mui/material/Link';
+import React, { useCallback, useState } from 'react';
+import { Link as ReactRouterLink } from 'react-router-dom';
+
+import { ActionButtonContainer } from 'components/common/ActionButtonContainer';
+import InputDialog from 'components/common/InputDialog';
+import { FapReviewTemplate, Review, TemplateGroupId } from 'generated/sdk';
+import { useFormattedDateTime } from 'hooks/admin/useFormattedDateTime';
+import { useCallsData } from 'hooks/call/useCallsData';
+import { useReviewsData } from 'hooks/review/useReviewsData';
+import { tableIcons } from 'utils/materialIcons';
+
+import TemplatesTable, { TemplateRowDataType } from './TemplatesTable';
+
+function CallsList(props: { filterTemplateId: number }) {
+  const { calls } = useCallsData({ templateIds: [props.filterTemplateId] });
+  const { toFormattedDateTime, timezone } = useFormattedDateTime({
+    shouldUseTimeZone: true,
+  });
+
+  const callListColumns = [
+    { title: 'Short Code', field: 'shortCode' },
+    {
+      title: `Start Date(${timezone})`,
+      field: 'startCallFormatted',
+    },
+    {
+      title: `End Date(${timezone})`,
+      field: 'endCallFormatted',
+    },
+  ];
+
+  const callsWithFormattedDates = calls.map((call) => ({
+    ...call,
+    startCallFormatted: toFormattedDateTime(call.startCall),
+    endCallFormatted: toFormattedDateTime(call.endCall),
+  }));
+
+  return (
+    <MaterialTable
+      icons={tableIcons}
+      title="Calls"
+      columns={callListColumns}
+      data={callsWithFormattedDates}
+    />
+  );
+}
+
+function FapReviewsList(props: { filterTemplateId: number }) {
+  const { reviewsData } = useReviewsData({
+    templateIds: [props.filterTemplateId],
+  });
+
+  const reviewListColumns = [
+    {
+      title: 'Review ID',
+      field: 'reviewId',
+      render: (review: Review) => (
+        <ReactRouterLink to={`/Reviews?reviewModal=${review.id}`}>
+          {review.id}
+        </ReactRouterLink>
+      ),
+    },
+    {
+      title: `Grade`,
+      field: 'grade',
+    },
+    {
+      title: `Status`,
+      field: 'status',
+      render: (review: Review) => review.status,
+    },
+  ];
+
+  return (
+    <MaterialTable
+      icons={tableIcons}
```

Comment:
Same here with loading state of the reviews and the table. `isLoading` on table can be connected to `loading` from `useReviewsData`

Reply:
Done